install/0000_30_machine-api-operator_01_images.configmap.yaml

```diff
@@ -12,5 +12,11 @@ data:
       "clusterAPIControllerLibvirt": "docker.io/openshift/origin-libvirt-machine-controllers:v4.0.0",
       "clusterAPIControllerBareMetal": "quay.io/openshift/origin-baremetal-machine-controllers:v4.0.0",
       "clusterAPIControllerAzure": "quay.io/openshift/origin-azure-machine-controllers:v4.0.0",
-      "clusterAPIControllerGCP": "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0"
+      "clusterAPIControllerGCP": "quay.io/openshift/origin-gcp-machine-controllers:v4.0.0",
+      "baremetalOperator": "quay.io/metal3-io/origin-baremetal-operator:latest",
+      "baremetalProvider": "quay.io/metalkube/metalkube-ironic",
```

Comment:
why master vs latest, can we use latest all the time? note this is used as a place holder at the time of building the payload to be replaced by the imageStream matching these names in image-references